projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts

```diff
@@ -166,6 +166,66 @@ export const tabbingOrderConfig: TabbingOrderConfig = {
     { value: 'Submit', type: TabbingOrderTypes.BUTTON },
     { value: 'Cancel', type: TabbingOrderTypes.BUTTON },
   ],
+  productConfigurationPage: [
```

Comment:
Could you explain what it is?

Reply:
this is the configuration file, in which you definde the "tabbing order", that is the order in which you can tab through the page. The file is used by method verifyTabbingOrder.